doc/class_property_descriptor.md

```diff
@@ -6,6 +6,88 @@ specific to each `Napi::ObjectWrap<T>` subclass.
 This prevents using descriptors from a different class when defining a new class
 (preventing the callbacks from having incorrect `this` pointers).
 
+## Example
```

Comment:
Nit: @ross-weir would you mind fixing the typo a few lines above this line? It reads "# Class propertry and descriptor" whereas it should read "#Class property and descriptor".

Reply:
Sure thing. Fixed